README.md

```diff
@@ -68,12 +88,15 @@ public final class DeviceAdvertiser: NSObject {
 }
 
 extension DeviceAdvertiser: MCNearbyServiceAdvertiserDelegate {
-    public func advertiser(_ advertiser: MCNearbyServiceAdvertiser, 
-                            didReceiveInvitationFromPeer peerID: MCPeerID, 
-                            withContext context: Data?, 
-                            invitationHandler: @escaping (Bool, MCSession?) -> Void) {
+    public func advertiser(_ advertiser: MCNearbyServiceAdvertiser,
+                           didReceiveInvitationFromPeer peerID: MCPeerID,
+                           withContext context: Data?,
+                           invitationHandler: @escaping (Bool, MCSession?) -> Void) {
         invitationHandler(false, MCSession())
     }
+    
+    public func advertiser(_ advertiser: MCNearbyServiceAdvertiser, didNotStartAdvertisingPeer error: Error) {
+    }
```

Comment:
Note: It looks useless but this function was added in order to avoid an unexpected crash on iOS 14.
`App crashes when Register button is tapped from Push Screen on iOS 14` (Demo app)